style.css

```diff
@@ -356,3 +342,63 @@ img {
   background: #6070ff;
   color: white;
 }
+
+.resumeBtn:active {
+  background: #2230d2;
+  color: white;
+}
+
+.resumeBtn:disabled {
+  background: white;
+  color: #5e6c84;
+}
+
+/* Language */
+
+.languages {
+  margin-top: 30px;
+  background: #fff;
+  padding-bottom: 40px;
+}
+
+.language {
+  border-bottom: none !important;
+}
+
+.language-wrapper .language,
+.frame,
+.skills {
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Thanks will look into it.